identity/app/views/layout/identityHeader.scala.html

```diff
@@ -1,47 +1,53 @@
 @import common.LinkTo
 @import views.support.DropdownMenus.accountDropdownMenu
 
-@()(implicit page: model.Page, request: RequestHeader)
+@(hideNavigation:Boolean = false)(implicit page: model.Page, request: RequestHeader)
 
 <header class="identity-header" data-component="identity-nav">
     <div class="gs-container identity-header__container">
 
-        <div class="identity-header__links">
-            <div class="identity-header__link">
-                <label for="identity-header-account-menu-fallback" tabindex="0">
-                    <span
+        @if(!hideNavigation) {
+            <div class="identity-header__links">
+                <div class="identity-header__link">
+                    <label for="identity-header-account-menu-fallback" tabindex="0">
+                        <span
                         role="button"
                         class="identity-header__nav js_identity-header__nav-toggle"
                         aria-expanded="false"
                         aria-controls="my-account-dropdown"
                         data-link-name="identity-nav : menu : toggle"
-                    >
-                        @fragments.inlineSvg("profile-36", "icon", List("mobile-only"))
-                        <span class="hide-on-mobile-inline">My Account</span>
-                    </span>
-                </label>
-                <input type="checkbox" value="open" id="identity-header-account-menu-fallback" class="identity-header__nav-fallback u-h" title="Open Menu" />
-                <ul class="identity-dropdown" id="my-account-dropdown" aria-hidden="true">
+                        >
+                            @fragments.inlineSvg("profile-36", "icon", List("mobile-only"))
+                            <span class="hide-on-mobile-inline">My Account</span>
+                        </span>
+                    </label>
+                    <input type="checkbox" value="open" id="identity-header-account-menu-fallback" class="identity-header__nav-fallback u-h" title="Open Menu" />
+                    <ul class="identity-dropdown" id="my-account-dropdown" aria-hidden="true">
                     @for((item) <- accountDropdownMenu) {
```

Comment:
is the formatting inside this `for` intended?

Reply:
nope! nice catch